Examples/Swift/AppDelegate.swift

```diff
@@ -13,7 +13,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
It would be nice to add a compatibility alias so that the code would still compile with older Xcode

Something like:
```
#if !swift(>=4.2)
extension UIApplication {
  static let LaunchOptionKey = UIApplicationLaunchOptionKey
}
#endif
```